app/controllers/hiring_staff/sign_in/dfe/sessions_controller.rb

```diff
@@ -23,8 +23,8 @@ def create
 
   private
 
-  def update_session(school_urn)
-    session.update(session_id: oid, urn: school_urn)
+  def update_session(school_urn, permissions)
+    session.update(session_id: oid, urn: school_urn, tva_permissions: permissions.all_permissions)
```

Comment:
Given we are no longer rendering our own page for them to select options I wonder if a basic `yes` or `no` as the session value would now be enough to make this work and make it clearer to the reader.